spec/filters/grok_spec.rb

```diff
@@ -1,5 +1,13 @@
 # encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+
+# running the grok code outside a logstash package means
+# LOGSTASH_HOME will not be defined, so let's set it here
+# before requiring the grok filter
+unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
+  LogStash::Environment::LOGSTASH_HOME = File.expand_path("../../../", __FILE__)
```

Comment:
aww right, of course. I usually do const_defined? with a string, `const_defined?("LOGSTASH_HOME")` .... so if it also works with symbols, all good.